docs/_spec/openapi-split.yaml

```
@@ -6,41 +6,51 @@ info:
     url: "images/TROLIE-horizontal-color.svg"
   description: |
 
-    Pursuant to FERC Order 881, Transmission Providers are required to support
-    Ambient Adjusted Ratings (AARs) for most Transmission Facilities in their
-    footprint.  In this context, Transmission Providers refers roughly to the
-    FERC definition of Transmission Provider as any public utility that owns,
-    operates, or controls facilities used for the transmission of electric
-    energy in interstate commerce.  This could be said to apply to any entity
-    that operates the transmission grid, including ISOs.
-    
-    For the purpose of this API, Transmission Provider may be assumed to be more
-    generic- it is really any entity that can receive AARs.  This API
-    specification supports the information exchange necessary to coordinate
-    forecasted and base facility ratings between a Transmission Provider and
-    Ratings Providers.  The Transmission Provider is assumed to be the host of
-    this information exchange.
-
-    A Ratings Provider is defined by this specification to be any entity that
-    has pre-coordinated with the Transmission Provider hosting this information
-    exchange to be the entity responsible for providing AARs on some set of
-    Transmission Facilities known to the Transmission Provider. That
-    pre-coordination is out-of-scope for this specification.  The language of
-    this API makes a strict distinction between the terms rating and limit.
-
-    A limit is a validated value that is actively used to make operational
-    decisions on the power grid.  This could be limits used by the EMS or
-    real-time market in real-time, or it could be forecast limits used by
-    look-ahead processes such as EMS study applications or the day-ahead market.
-
-    A rating, in contrast, is a calculated value from some process that is input
-    to TROLIE (and by extension operating power systems at large) and may be
-    selected for use as a limit after validation and dependent on operating
-    conditions, time frame and the operational decisions being made.  Seasonal
-    ratings, forecast rating proposals, and real-time rating proposals are all
-    examples of ratings.
-
-    <img src="images/TROLIE Ratings Provider interactions.excalidraw.png" alt="Primary Interactions with Ratings Provider" />
+    This specification defines a set of operations for the exchange of power
+    system ratings and limits between entities that own and operate the electric
+    power system in North America. In particular, it is designed to support the
+    exchange of Ambient Adjusted Ratings (AARs), pursuant to FERC Order 881.  It
+    is being promulgated as a community standard to facilitate interoperability
```

```suggestion
    is published as a community standard to facilitate interoperability
```
Nobody knows what promulgated means.  I had to look it up.  I know my suggestion is less precise and perhaps a little more presumptuous than your wording, but I suggest it is probably enough for the audience.